__tests__/snapshot.js

```diff
@@ -0,0 +1,7 @@
+import { render } from '@testing-library/react'
+import Home from '../pages/index'
+
+it('renders homepage unchanged', () => {
+  const { container } = render(<Home />)
+  expect(container).toMatchSnapshot()
+})
```

Comment:
Nice to have a snapshot test as an example but we don't need it as our regular testing. Unit test is the only one for now.

You can keep this file BTW.